src/appkit.cs

```diff
@@ -18186,6 +18708,141 @@ interface NSTouch : NSCopying {
 		CGSize DeviceSize { get; }
 	}
 
+	[Mac (10, 12, 1)]
+	[Category]
+	[BaseType (typeof(NSTouch))]
+	interface NSTouch_NSTouchBar
+	{
+		[Export ("type")]
+		NSTouchType GetType ();
```

Comment:
Don't love this since it collides w\ GetType on Object, but not sure better name.